index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title></title>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+    <body>
+        <div id="page-wrapper">
+            <header id="header">
+              <div class="logo">
+                <img
+                  id="header-img"
+                  src="https://s3.amazonaws.com/freecodecamp/original_trombones.png"
+                  alt="original trombones logo"
+                />
+              </div>
+
+              <nav id="nav-bar">
```

Comment:
Only use id tag when you cant avoid it according to HTML best practices. Instead use semantic tag or class when necessary.